modules/cluster-logging-log-forwarding-configure.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-log-forwarding.adoc
+
+[id="cluster-logging-log-forwarding-configure_{context}"]
+= Configuring log forwarding
+
+To configure log forwarding, edit the Cluster Logging Custom Resource (CR) to enable forwarding and create a Log Forwarding Custom Resource to specify the outputs, pipelines, and enable log forwarding. 
+
+[NOTE]
+====
+Enabling the log forwarding feature disables the xref:../../logging/config/cluster-logging-external.adoc#cluster-logging-external[default log forwarding feature].
+====
+
+If you enable log forwarding, any undefined source types will be dropped. For example, if you specified a pipeline for the `logs.app` 
```

Comment:
May wish to mention this kind of info is available from the status with why

Reply:
@jcantrill Which status shows this info? 

@QiaolingTang Do you have an example of this output?

Reply:
@mburke5678 Is this what you need? http://pastebin.test.redhat.com/816604

Reply:
@QiaolingTang Thank you. However, @jcantrill mentioned previously _this kind of info is available from the status with why_. I am hoping to see that status message.

Reply:
@mburke5678 there has `- reason: UnrecognizedSourceType` in the status field, you can check it in my last pastebin.

Reply:
@QiaolingTang I apologize. I didn't look for the status section of your pastebin. 
Is there a similar message if the user doesn't create a pipeline for one of the log source types?

Reply:
Status info starts here https://github.com/openshift/cluster-logging-operator/blob/master/pkg/apis/logging/v1alpha1/forwarding_types.go#L122 and set here https://github.com/openshift/cluster-logging-operator/blob/master/pkg/k8shandler/forwarding.go#L55

Reply:
@mburke5678 
> Is there a similar message if the user doesn't create a pipeline for one of the log source types?

I think in this condition there won't have any messages, https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/cluster-logging-log-forwarding.md#log-message-routing.

Reply:
@QiaolingTang Sorry for the confusion. Is there any information in the Status fields if the user does not provide a pipeline for a source? Here there is no `inputSource: logs.app`. Would there be a message that the source is missing?

````
  pipelines:
   - name: infra-logs
     inputSource: logs.infra:
     outputRefs:
     - elasticsearch-insecure
   - name: audit-logs
     inputSource: logs.audit
     outputRefs:
     - secureforward-offcluster
````

Reply:
No, as it is said in https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/cluster-logging-log-forwarding.md#log-message-routing:  `For example, If you have specified a pipeline for logs.app sources, but did not specify one for logs.infra, then logs.infra will be dropped`, if there is no `inputSource: logs.app`, the app logs will be droped.